src/endless/EndlessUsbToolDlg.cpp

```diff
@@ -5080,6 +5080,8 @@ bool CEndlessUsbToolDlg::WriteMBRAndSBRToWinDrive(CEndlessUsbToolDlg *dlg, const
 		}
 	}
 
+	dlg->eventTracking(_T("BootTrackSizeKB"), _T(""), minStartingOffset / 1024);
```

Comment:
I think we should log this in bytes, not KB. Rationale: the size is a multiple of the sector size, which is 512.